sdk/communication/Azure.Communication.Email/samples/Sample1_SendSimpleEmailWithAutomaticPollingForStatus.md

```diff
@@ -37,4 +37,4 @@ catch ( RequestFailedException ex )
 }
 ```
 
-[README]: https://learn.microsoft.com/azure/communication-services/quickstarts/email/send-email?tabs=windows&pivots=platform-azcli#prerequisites
+[README]: https://learn.microsoft.com/azure/communication-services/quickstarts/email/send-email?tabs=windows&pivots=platform-azcli#prerequisites
```

Comment:
what changed in this and other sample files?

Reply:
These are all autogenerated by the samples script. I'm not entirely sure why Github is picking up a diff here, but the linking isn't affected.